frontend/public/components/utils/details-item.tsx

```diff
@@ -39,7 +39,7 @@ export const PropertyPath: React.FC<{ kind: string; path: string | string[] }> =
 const EditButton: React.SFC<{ onClick: (e: React.MouseEvent<HTMLButtonElement>) => void }> = (
```

Comment:
```suggestion
const EditButton: React.FC<{ onClick: (e: React.MouseEvent<HTMLButtonElement>) => void, id: string }> = (
```

Reply:
extracted out the props to a `type EditButtonProps = { }`